metrics/storage/blogbench_dockerfile/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # Set up an Ubuntu image with 'blogbench' installed
 
 # Usage: FROM [image name]
-FROM ubuntu
+FROM public.ecr.aws/lts/ubuntu
```

Comment:
@Amulyam24 I once [promised](https://github.com/kata-containers/tests/pull/3607#pullrequestreview-678529665) @fidencio to only use `quay.io/libpod/ubuntu` when moving from `docker.io` ever more. Could we make that consistent here? :)

Reply:
@Jakob-Naucke , having `libpod/ubuntu` image was my initial change. However, as per @devimc's suggestion [here](https://github.com/kata-containers/tests/pull/3700#discussion_r667907019), I moved it to the current image.

Reply:
ah, sorry. Keep it then, I'll look into updating elsewhere.

Reply:
Yep, this is fine as libpod folks advertised @devimc, IIRC, to not use that image as it's outdated, and it seems I forgot to forward the message. :-/